sds_data_manager/lambda_code/SDSCode/pipeline_lambdas/batch_starter.py

```diff
@@ -1273,10 +1276,20 @@ def lambda_handler(events: dict, context):
         Lambda context object
     """
     logger.info(f"Events: {events}")
+    is_auth_user = is_authenticated_path(events)
+    is_reprocessing = (
+        api_event.get("reprocessing")
+        if (api_event := events.get("queryStringParameters"))
+        else False
+    )
```

Comment:
Use the default get return to chain these together.
```suggestion
    is_reprocessing = events.get("queryStringParameters", {}).get("reprocessing", False)
```